source/MaterialXGenMdl/MdlSyntax.cpp

```diff
@@ -37,13 +37,19 @@ class MdlFilenameTypeSyntax : public ScalarTypeSyntax
 
     string getValue(const Value& value, bool /*uniform*/) const override
     {
+        const string outputValue = value.getValueString();
```

Comment:
Avoid invalid syntax of create a texture_2d() lookup using "/" or "" as the input. Just use default = "texture_2d()".